lib/wx/src/gen/wxEvtHandler.erl

```diff
@@ -36,7 +36,7 @@
 %% Beware that the callback will be in executed in new process each time.
 %%
 %% <a href="http://www.wxwidgets.org/manuals/stable/wx_wxevthandler.html">
-%% The orginal documentation</a>. 
+%% The original documentation</a>. 
```

Comment:
This is also generated, have generated this or edited the file because I don't want changes when I generate next time.